UnityProject/Assets/Scripts/Items/IDCard.cs

```diff
@@ -213,6 +222,10 @@ public void SyncIDCardType(IDCardType oldCardType, IDCardType cardType)
 	/// <returns></returns>
 	public bool HasAccess(Access access)
```

Comment:
wait a second, what the fuck is this method. What uses it? I think most of stuff uses https://github.com/unitystation/unitystation/blob/develop/UnityProject/Assets/Scripts/Systems/Occupations/AccessRestrictions.cs this one to check for access